sigmadsp/communication/tcpip1701.py

```diff
@@ -0,0 +1,186 @@
+"""This module communicates with SigmaStudio.
```

Comment:
this module's file name does not match the scheme you chose for `tcpipadau145x.py`

I would prefer `tcpip_adau145x.py` and `tcpip_adau1701.py` or even `requests_handling_adau145x.py` and `requests_handling_adau1701.py`.

Reply:
I used the names of the protocol blocks in sigma studio; also, something like `requests_handling_adau145x` is pointlessly verbose IMO

Reply:
ah, I remembered wrong, nevermind; will gather the common parts in a parent class